LibreHardwareHelper/LibreHardwareHelper/Model/HardwareData/CPU/CpuTemp.cs

```diff
@@ -44,7 +44,7 @@ public CpuTemp(IHardware cpu, LibreHardwareHelper helper)
                     CoreAverage = s.Value ?? 0;
                     break;
                 }
-                case "Core (Tct1/Tdie)":
+                case "Core (Tctl/Tdie)":
```

Comment:
Use case fall through since the assignment is already being made for CoreAverage